conda-forge.yml

```diff
@@ -1,8 +1,9 @@
+azure:
+  store_build_artifacts: true
```

Comment:
Was this left in intentionally?

Reply:
No :(

Reply:
Fixed in https://github.com/conda-forge/ctng-compiler-activation-feedstock/commit/9d65b2e5d051628c2b604fe54105ba76cc57927d so we don't forget next time around.